src/areas/Login/FirebaseConfig.js

```diff
@@ -0,0 +1,30 @@
+import firebase, {initializeApp} from 'firebase/compat/app';
+import {getAuth} from 'firebase/auth'
+
+export const firebaseConfig = {
```

Comment:
Ok cool. Wasn't sure where to put it, so just kept it in the login folder since that was the closest relation to it. Will move it out to the separate folder.